docs/templates/eia861_child.rst.jinja

```diff
@@ -18,12 +28,36 @@ details, see the `official EIA-861 page <{{ source.path }}>`__
 {% endblock %}
 
 {% block availability %}
+PUDL incorporates all available EIA-861 data (except the 2019 data) dating back to 2001.
+Earlier data from the forms EIA-861 (utility, Sales to Ultimate Customers, and Operational data),
+and EIA-861S (Sales to Ultimate Customers) exists for the years 1990-2000, but is not yet included in PUDL.
+As of 2023, PUDL only incorporates the annual data, but there's also a less detailed
+monthly `Form EIA-861M <https: //www.eia.gov/electricity/data/eia861m />`__.
```

Comment:
I would just say: "There is also a less detailed monthly....[your text].. that is not incorporated into PUDL."